lib/eldap/src/eldap.erl

```diff
@@ -36,6 +36,54 @@ Type definitions that are used more than once in this module:
 - **`referrals() =`** - `[Address = string()]` The contents of `Address` is
   server dependent.
 """.
+
+-type connection_info() :: #{socket := ssl:sslsocket() | gen_tcp:socket(), socket_type := tcp | ssl}.
+-type search_option() ::
+    {base, string()} |
+    {filter, filter()} |
+    {scope, scope()} |
+    {attributes, [string()]} |
+    {deref, dereference()} |
+    {types_only, boolean()} |
+    {timeout, integer()}.
+
+-doc "handle() - An opaque handle unique for the connection".
+-type handle() :: term().
+
+-doc "attribute() - `{Type = string(), Values=[string()]}`".
```

Comment:
Since the doc string is basically identical to the type, I think you can remove it.

Reply:
yes, let me fix it.